docs/source/tutorials/clean_learning/index.rst

```diff
@@ -0,0 +1,9 @@
+CleanLearning Tutorials
+=================
```

Comment:
The underline should extend at least as far as the right edge of the title ([link](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections)).

```suggestion
CleanLearning Tutorials
=======================
```